apps/client/src/app/app.component.ts

```diff
@@ -208,4 +210,10 @@ export class AppComponent implements OnDestroy, OnInit {
       this.document.body.classList.remove('is-dark-theme');
     }
   }
+
+  private toggleMetaThemeColor(isDarkTheme: boolean) {
+    const color = isDarkTheme ? 'rgb(25, 25, 25)' : 'rgb(255, 255, 255)';
```

Comment:
Thanks the getBackgroundColor seems to have some hidden check that prevents this to work correctly. I used the getCssVariable directly so that it can change when the user changes the setting.
Interesting to check if the places that use getBackgroundColor are expecting it to not change if the user changed the theme to light but has "prefers-color-scheme: dark".